replica/distributed_loader.cc

```diff
@@ -198,9 +198,8 @@ distributed_loader::process_upload_dir(distributed<replica::database>& db, shard
 
         auto make_sstable = [&] (shard_id shard) {
             auto& sstm = global_table->get_sstables_manager();
-            bool uuid_sstable_identifiers = sstm.uuid_sstable_identifiers();
-            auto generation = sharded_gen.invoke_on(shard, [uuid_sstable_identifiers] (auto& gen) {
-                return gen(sstables::uuid_identifiers{uuid_sstable_identifiers});
+            auto generation = sharded_gen.invoke_on(shard, [] (auto& gen) {
+                return gen();
```

Comment:
Is this hunk related?

Reply:
Hmm, looks like a bad squash.
Should be in "sstables: sstable_generation: get rid of uuid_identifiers bool class"